### ISSUE TITLE
Crash when clicking on ViewEvent source actions

### ISSUE THREAD

**Issue by bmarty** [element-hq/element-android#4279]
Probably due to the migration to MvRx2, see the stack trace:

```
    java.lang.NoSuchMethodError: No direct method <init>(Landroidx/lifecycle/LifecycleOwner;Lkotlin/jvm/functions/Function0;)V in class Lcom/airbnb/mvrx/lifecycleAwareLazy; or its super classes (declaration of 'com.airbnb.mvrx.lifecycleAwareLazy' appears in /data/app/im.vector.app.debug-czukF-LVif-ZqSrtYwRdEw==/base.apk)
        at org.billcarsonfr.jsonviewer.JSonViewerFragment.<init>(JSonViewerFragment.kt:88)
        at org.billcarsonfr.jsonviewer.JSonViewerFragment$Companion.newInstance(JSonViewerFragment.kt:71)
        at org.billcarsonfr.jsonviewer.JSonViewerDialog.onActivityCreated(JSonViewerDialog.kt:29)
        at androidx.fragment.app.Fragment.performActivityCreated(Fragment.java:2996)
```

- [x] Fix the issue, may have to be fixed on https://github.com/BillCarsonFr/JsonViewer
- [x] #4308

**Comment by bmarty** [comment]
@BillCarsonFr will update his lib org.billcarsonfr.jsonviewer

**Comment by ganfra** [comment]
Adding test will be done on another PR, along fixing the localhost issue for synapse config.

**Comment by bmarty** [comment]
Crash fixed on 1.3.5

Remaining point converted to a dedicated issue
